pkg/broker/work_engine.go

```diff
@@ -7,32 +7,57 @@ type Work interface {
 	Run(token string, msgBuffer chan<- WorkMsg)
 }
 
+// WorkTopic - Topic jobs can publish messages to, and subscribers can listen to
+type WorkTopic string
```

Comment:
Can we keep the type declaration near the const definition? Will be easier for people reading godoc, or code, that it is supposed to be a kind of enum IMO.